src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import frc.robot.Constants;
+
+public class Intake extends SubsystemBase {
+    private CANSparkMax m_intakeMotor;
+    private Solenoid m_deploy;
+
+    public Intake() {
+        m_intakeMotor = new CANSparkMax(Constants.kIntakeMotorId, MotorType.kBrushless);
+        m_deploy = new Solenoid(PneumaticsModuleType.CTREPCM, Constants.kIntakeSolenoid);
+
+        m_intakeMotor.setSmartCurrentLimit(40);
+        m_intakeMotor.setIdleMode(CANSparkMax.IdleMode.kCoast);
+    }
+
+    public void lowerIntake() {
+        m_deploy.set(true);
+    }
+
+    public void raiseIntake() {
+        m_deploy.set(false);
+    }
+
+    public void startIntakeMotor(int direction, double speed) {
```

Comment:
I we should make a function called `spin` or something that takes no arguments. 

This `start*Motor` pattern we have I would argue is slightly breaking the abstraction of what a subsystem should be and a lot of the time we are using speed from constants defined in Constants.java. So we might as well make functions thats abstract that away from the commands.

Reply:
@KarlWheezer